specifyweb/frontend/js_src/lib/components/Toolbar/DataModel.tsx

```diff
@@ -234,8 +234,14 @@ function DataModelFields({
   readonly model: SpecifyModel;
 }): JSX.Element {
   const data = React.useMemo(() => getFields(model), [model]);
+  const scope = React.useMemo(
+    () => model.getScopingRelationship()?.relatedModel.name,
+    []
+  );
   return (
     <>
+      <p>Field ID: {model.idField.name}</p>
+      {scope && <p>Scope: {scope}</p>}
```

Comment:
Remember that this is bug-prone. Please refactor to use explicit check `typeof scope === 'string' && <p>...</p>`

You can also configure ESLint plugin to have it notify you of this automatically (https://github.com/specify/specify7/tree/production/specifyweb/frontend/js_src#plugins)